Equilobe.DailyReport.SL/ReportService.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Equilobe.DailyReport.DAL;
+using Equilobe.DailyReport.Models.Interfaces;
+using Equilobe.DailyReport.Models.Jira;
+using Equilobe.DailyReport.Models.ReportFrame;
+using Equilobe.DailyReport.Models.Storage;
+using System;
+using System.Linq;
+
+namespace Equilobe.DailyReport.SL
+{
+    public class ReportService : IReportService
+    {
+        public IJiraService JiraService { get; set; }
+
+        #region IReportService Implementation
+        public void UpdateDashboardData(long instanceId)
+        {
+            var jiraRequestContext = GetJiraRequestContext(instanceId);
+            var users = JiraService.GetInstanceUsers(jiraRequestContext)
+                .Select(p => ToAtlassianUser(p, instanceId))
+                .ToList();
+
+            using (var db = new ReportsDb())
+            {
+                var dbUsers = db.AtlassianUser.ToList();
```

Comment:
Here you update the dashboard for one instancem and you take from db all the users. Take only the users which have the instance id of the dashboard.

Reply:
You are right!